examples/testplans/release.json

```diff
@@ -9,12 +9,6 @@
 	{"name": "Avocado source does not contain spelling errors",
 	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make spell`. Expected result: Make command should say OK."},
 
-	{"name": "Avocado RPM build",
-	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/python-avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
-
-	{"name": "Avocado RPM lint",
```

Comment:
I haven't found any checks on copr with regards to warnings, will it fail? If not than we still need to perform this step on the downloaded package...

Reply:
I'm adding the `rpmlint` step to the machine where the packages are installed to avoid the duplicate download.